playbooks/install_stack.yaml

```diff
@@ -465,13 +465,24 @@
     become: true
     become_user: root
 
+  - name: Block to figure out if --standalone needs to be passed when deploying tripleo
+    when:
+    - ansible_facts.distribution == 'CentOS'
+    block:
+      - name: Check if the version of TripleO is recent enough to remove --standalone argument
+        set_fact:
+          standalone_toggle: false
+        when:
+        - tripleo_repos_branch is defined
+        - tripleo_repos_branch not in ['train', 'ussuri', 'victoria']
```

Comment:
oh I missed the case where tripleo_repos_branch is not defined (default, when using master).